internal/kldeth/txn.go

```diff
@@ -22,7 +22,8 @@ import (
 	"reflect"
 	"regexp"
 	"strconv"
-	"strings"
+  "strings"
```

Comment:
just a nit, this formatting looks wacky.

Reply:
this file probably needs a `gofmt` run

Reply:
I've run it through gofmt 👍